tools/clang/test/CodeGenHLSL/expressions/conversions_and_casts/numerical_const_init_list.hlsl

```diff
@@ -68,7 +68,7 @@ void main() {
     
     // To float
     // CHECK: float 1.000000e+00, float -1.000000e+00, float 0x41F0000000000000, float -1.000000e+00, i8
-    // CHECK: float 6.553500e+04, float -1.000000e+00, float 0x41F0000000000000, float -1.500000e+00, i8
+    // CHECK: float 6.553500e+04, float -1.000000e+00, float 0x43F0000000000000, float -1.500000e+00, i8
```

Comment:
Previously it was the float representation of `0xFFFFFFFF` rather than `0xFFFFFFFFFFFFFFFF`, I hadn't caught that while writing the test.